app/scripts/lib/country-telephone-info.js

```diff
@@ -84,6 +84,15 @@ define(function(require, exports, module) {
       prefix: '+43',
       rolloutRate: 1
     },
+    // Australia
+    // https://en.wikipedia.org/wiki/Telephone_numbers_in_Australia
+    AU: {
+      format: formatter('+61 ${serverPhoneNumber}'),
+      normalize: ensurePrefix('+61'),
+      pattern: /^(?:\+61\d{9}|\d{10})$/,
```

Comment:
Numbers in the 10-digit form should always have `0` as the first digit, like `04 1234 5678`.  If it's not a zero then it's some sort of weird telco dialing prefix that we definitely shouldn't be trying to message.

Reply:
@rfk - every country where the 0 trunk prefix is used is the same. We used to check for the `0` on the content server but removed the check in favor of leaning on auth server validation.

Reply:
OK, sounds good :-)